### DISCUSSION TITLE
有没有可能增加一个备案信息显示

### DISCUSSION THREAD

**Discussion by iamckun** [linuxserver/Heimdall#1001] [Q&A] [answered]

**Comment by ghost** [comment]
Translation for your favor:
Is it possible to add “Chinese official registration ID" in the page?
Example:
![图片](https://user-images.githubusercontent.com/77554817/202203047-300d88c4-02be-4114-a29a-5bad8afd40ab.png)


Note to you:
It is a general manner of politeness to add issue in English in Github. If you do not know how to use English, use Baidu translator or Google translator.
在Github开ticket用英文是基本礼仪，如果你不知道怎么用英文，请使用百度翻译或者谷歌翻译。

**Comment by KodeStar** [comment]
I have no idea what that means?

**Comment by ghost** [comment]
This is a shit thing from China. The Chinese law requires all public facing website in China (including personal hosted websites) to be registered and show their registration ID at the bottom of the homepage. This registration is only open for Chinese citizens and requires the verification of official personal ID.

That is why every single foreign (outside China) website is officially illegal in China and the Chinese government has right given by law to ban them via any technical methods. Anyone outside China who want to open a registered website in China has to find a Chinese citizen to back them up for the registration. In this way, the government can control all webs or at least know who is responsible for spreading "illegal" information. 

I would say if the maintainer disagrees with what I described above, simply close this ticket.

**Comment by KodeStar** [comment]
Thanks for the information @J4gQBqqR 

This is doable with some CSS, so just go to Settings / Custom CSS and add:

```
main#main:after {
    content: "Registration: 01234567";
    color: white;
}
```

**Comment by dovela** [comment, marked as answer]
摸索出来了，自定义js添加(Custom JavaScript add)
```
$("#main").append('<a style="font-size: 15px;color: white;text-align: center;"  href="https://beian.miit.gov.cn/" target="_blank">ICP备xxxxxxxx号</a>');
```

**Reply by iamckun** [reply]
谢谢，我已经弄好了就是代码多了一点
